cognify/optimizer/core/flow.py

```diff
@@ -129,6 +129,7 @@ class OptConfig:
     param_save_path: str = field(default=None)
     frugal_eval_cost: bool = field(default=True)
     use_SH_allocation: bool = field(default=False)
+    patience: tuple[float,float,int] = field(default=(0.01,0.01,5))
```

Comment:
There should be some bounds-checking for the actual values since `OptConfig` is exposed to the creator of the search function. This is actually a broader issue and doesn't just apply to `patience`, so I created an issue here: #21 . Doesn't have to be fixed by this PR, but we can start by at least adding bounds-checking for patience here.

Reply:
added